src/collective/easyform/actions.py

```diff
@@ -513,6 +514,13 @@ def __init__(self, **kw):
         super(CustomScript, self).__init__(**kw)
 
     def getScript(self, context):
+        # for the convenience of scripters operating
+        # in a restricted python environment,
+        # let's store a reference to FORM_ERROR_MARKER
+        # on the object, so it'll be available
+        # as an attribute of context.
+        context.FORM_ERROR_MARKER = FORM_ERROR_MARKER
```

Comment:
`context` here is the easyform object. This means that every time someone fills in the form, a database change is made. Maybe make it available on the request instead?

    context.REQUEST.FORM_ERROR_MARKER = FORM_ERROR_MARKER

Reply:
I thought about this as well, but was not sure and did not check whether this results in a DB change each time. We certainly should not do that. Sorry.

I suggested and did the change at the context, as this is how the FORM_ERROR_MARKER is done at PFG. OTOH I am fine to use the request instead. Let me quickly update my fork accordingly ...